package.json

```diff
@@ -25,6 +25,7 @@
     "ios/RNZcash-Bridging-Header.h",
     "ios/RNZcash.xcodeproj",
     "ios/RNZcash.xcworkspace",
+    "scripts/copyCheckpoints.js",
```

Comment:
If a script is meant to be runnable by the client app, then see this documentation: https://docs.npmjs.com/cli/v6/configuring-npm/package-json#bin

If this is done, then the user can run the command by name straight from their "scripts" without mentioning the node_modules path. This is how tools like "webpack" make themselves easily runnable.

Maybe it doesn't matter, though, since this step will go away soon with the new SDK upgrade. In that case, we actually control the iOS package contents, so we can use the same approach as we already do for Android.

Reply:
Nice catch. I forgot about that feature. Done